packages/button/package.json

```diff
@@ -24,5 +24,8 @@
   },
   "publishConfig": {
     "access": "public"
+  },
+  "devDependencies": {
+    "@types/react": "^16.8.19"
```

Comment:
should this match the react version? (16.4.2) Or is it ok to have the newer types involved?

Reply:
A ton of unit tests typings are failing - might be related to versioning